frontend/src/metabase/components/TokenField.jsx

```diff
@@ -602,22 +601,15 @@ const DefaultTokenFieldLayout = ({
   isFocused,
   onClose,
 }) => (
-  <OnClickOutsideWrapper handleDismissal={onClose}>
-    <div>
-      {valuesList}
-      <Popover
-        isOpen={isFocused && !!optionsList}
-        hasArrow={false}
-        tetherOptions={{
-          attachment: "top left",
-          targetAttachment: "bottom left",
-          targetOffset: "10 0",
-        }}
```

Comment:
Replaced with `placement`. Matching `targetOffset` felt unnecessary here; the default is `[0, 5]`, and that looks absolutely fine (check out the recording).